### PR TITLE
Local Sandbox with Docker Compose.

### PR DESCRIPTION
**Pull request by ifursova**
Local Sandbox setup with the `docker-compose`. 

`sandbox.yml` - docker compose config
`kafka.env` - kafka settings

`demoContainer` - temp container for research purposes (`Ubuntu`, `bash`, `sudo`, `iproute2`, `iputils-ping`)

Links from the Demo:
[Docker Container Networking](https://platform9.com/blog/container-namespaces-deep-dive-container-networking/)
[Docker Compose Networking](https://docs.docker.com/compose/networking/)

### CODE REVIEW

File: sandbox.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      ZOOKEEPER_CLIENT_PORT: 2181
+
+  kafka:
+    image: confluentinc/cp-kafka:7.0.1@sha256:dee82c02cd7d36cea8d3369f181f47412adecf1001c10bbd4b4959610868dff5
+    container_name: kafka
+    depends_on:
+      - zookeeper
+    networks:
+      - kafka_test_network
+    ports:
```
Comment (borisenko-i) [inline sandbox.yml:18]: Is it possible to replace `ports` with `expose`?
https://docs.docker.com/compose/compose-file/compose-file-v3/#expose

File: sandbox.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
+version: '2'
```
Comment (borisenko-i) [inline sandbox.yml]: Maybe use version 3?

**Review by borisenko-i — COMMENTED** [review]
@ifursova ~~I've made a few changes to fix the build, could you take a look please? Those are a kind of ugly and I'm sure there's a better way to do this, but it's the best I could do to make the project build on my machine. If it works for you, we can keep it this way for now to avoid spending too much time on Gradle. However, if you know how to optimize this, feel free to make changes.~~

_EDIT: sorry, disregard the above. Right after pushing and writing this comment, I tried to rollback to your changes, then clean and rebuild again, and everything worked. It turns out the only issue was my older Gradle version 🤦‍♂️_ 

Also, could you add a sentence or two to the readme about how you tested the apps running in the sandbox?

And below are two minor things about the docker compose file.

File: sandbox.yml (hunk 1 above)
Comment (ifursova) [reply sandbox.yml:18]: The `ports` configuration is used only for the `kafka` container and it publishes the `29093` port to the host machine. I think for the debug purpose it is pretty convenient (it easier to run container than to start ZK and Kafka locally). The `expose` allows only internal network availability. 
I would keep `ports` here.

File: sandbox.yml (hunk 2 above)
Comment (ifursova) [reply sandbox.yml]: Changed to '3.9'

**Pull request merged**